packages/abstractions/src/authentication/baseBearerTokenAuthenticationProvider.ts

```diff
@@ -35,4 +35,8 @@ export class BaseBearerTokenAuthenticationProvider implements AuthenticationProv
 			}
 		}
 	};
+
+	public get accessTokenProviderInstance(): AccessTokenProvider {
+		return this.accessTokenProvider;
+	}
```

Comment:
```suggestion
```
Sorry, because of TS syntax I didn't realize that the field was actually public readonly, in effects, we don't need a getter.